scripts/check_frontend_test_coverage_test.py

```diff
@@ -28,7 +27,6 @@
 
 
 class CheckFrontEndCoverageTests(test_utils.GenericTestBase):
```

Comment:
Bit of a nit, but it should be "Frontend" here and elsewhere, since "frontend" (like in the filename) is one word.

Reply:
Yes, done.